services/core-api/requirements.txt

```diff
@@ -25,6 +25,7 @@ requests==2.22.0
 six==1.12.0
 SQLAlchemy==1.3.5
 sqlalchemy-filters==0.10.0
+tuspy==0.2.4
```

Comment:
FunFact: This library is supported by the same group that supports our frontend tus client. 

https://github.com/tus/tus-py-client

Reply:
nice